### ISSUE TITLE
Label created notification

### ISSUE THREAD

**Issue by thekavikumar** [opencv/cvat#7636]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://opencv.github.io/cvat/docs/)

### Is your feature request related to a problem? Please describe.

When I was creating a project, I created a label but got confused about whether the label had been created or not and to verify it, I had to close the label editor and verify whether the label that I created was listed or not. I strongly feel that many other users may also get confused due to the same problem. 

### Describe the solution you'd like

My solution is to add a notification.success whenever a user creates a new label so that user will get the confirmation about the label creation status.

![image](https://github.com/opencv/cvat/assets/50093149/72dc510c-d8d2-412a-9b29-b5fbf88be644)

**Comment by thekavikumar** [comment]
@nmanovic please assign me this issue

**Comment by nmanovic** [comment]
@thekavikumar , could we solve the issue in another way? I believe the right approach here is to view the list of labels when a user adds them. A notification is fine, but the list of labels should be not hidden when you add a new one. What do you think?

**Comment by thekavikumar** [comment]
@nmanovic yeah that would be a good solution, but I have to modify the whole form UI right

**Comment by nmanovic** [comment]
@thekavikumar , please prepare a short mockup where we will see how the form will look like after changes.

**Comment by thekavikumar** [comment]
@nmanovic  sure, once I'm done will let you knw

**Comment by thekavikumar** [comment]
I've found another way, hope this works. check it out:
https://github.com/opencv/cvat/pull/7651

when ever the user creates a label the label constructor closes and the
created label will be visible to the user and he/she can create another one
by clicking add label.
